src/flask_debugtoolbar/templates/panels/sqlalchemy_error.html

```diff
@@ -20,7 +20,7 @@ <h5>Install required libraries:</h5>
     <h5>Configure Flask-SQLAlchemy:</h5>
     <p>
       The Flask-SQLAlchemy extension needs to be configured for this application.
-      Please see the <a href="https://flask-sqlalchemy.palletsprojects.com/en/master/quickstart/">
+      Please see the <a href="https://flask-sqlalchemy.palletsprojects.com/en/latest/quickstart/">
```

Comment:
Hi,

You can use https://flask-sqlalchemy.palletsprojects.com/quickstart/ which is slightly more portable since it doesn't force `en` and will auto-redirect to the latest stable release instead of the development version.

Reply:
Even better, thank you.